plugins/content_distribution/providers/tvinci/lib/xml/tvinci_default.xslt

```diff
@@ -579,104 +581,5 @@
-            <xsl:when test="$tag = $MBR_TAG">
-                <xsl:value-of select="$mbrPpvModule"/>
-            </xsl:when>
-            <xsl:when test="$tag = $IPHONE_TAG">
-                <xsl:value-of select="$iphonenewPpvModule"/>
-            </xsl:when>
-            <xsl:when test="$tag = $IPAD_TAG">
-                <xsl:value-of select="$ipadnewPpvModule"/>
-            </xsl:when>
-            <xsl:when test="$tag = $DASH_TAG">
-                <xsl:value-of select="$dashPpvModule"/>
-            </xsl:when>
-            <xsl:when test="$tag = $WIDEVINE_TAG">
-                <xsl:value-of select="$widevinePpvModule"/>
-            </xsl:when>
-            <xsl:when test="$tag = $WIDEVINE_MBR_TAG">
-                <xsl:value-of select="$widevineMbrPpvModule"/>
-            </xsl:when>
-        </xsl:choose>
-    </xsl:template>
-    <!-- get the matching tag file name - which is given as argument to this code -->
-    <xsl:template name="type-name-matching-tag">
-        <xsl:param name="tag"/>
-        <xsl:param name="isChild"/>
-        <xsl:param name="childNumber"/>
-        <xsl:variable name="configuredName">
-            <xsl:choose>
-                <xsl:when test="$tag = $ISM_TAG">
-                    <xsl:value-of select="$ismTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $MBR_TAG">
-                    <xsl:value-of select="$mbrTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $IPHONE_TAG">
-                    <xsl:value-of select="$iphonenewTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $IPAD_TAG">
-                    <xsl:value-of select="$ipadnewTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $DASH_TAG">
-                    <xsl:value-of select="$dashTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $WIDEVINE_TAG">
-                    <xsl:value-of select="$widevineTypeName"/>
-                </xsl:when>
-                <xsl:when test="$tag = $WIDEVINE_MBR_TAG">
-                    <xsl:value-of select="$widevineMbrTypeName"/>
-                </xsl:when>
-            </xsl:choose>
-        </xsl:variable>
-        <xsl:variable name="nameSuffix">
-            <xsl:if test="$isChild">
-                <xsl:variable name="trailerSuffix">
-                    <xsl:value-of select="concat(' ',$CONST_TRAILER_NAME)"/>
-                </xsl:variable>
-                <xsl:variable name="trailerIdxSuffix">
-                    <xsl:if test="(($childNumber > 1))">
-                        <xsl:value-of select="concat(' ',$childNumber)"/>
-                    </xsl:if>
-                </xsl:variable>
-                <xsl:value-of select="concat($trailerSuffix, $trailerIdxSuffix)"/>
-            </xsl:if>
-        </xsl:variable>
-        <xsl:value-of select="concat($configuredName, $nameSuffix)"/>
-    </xsl:template>
 
 </xsl:stylesheet>
```

Comment:
Did you check the case when you put two 'same tag yet different' configurations and got two different files ?